create-openssl-framework.sh

```diff
@@ -151,7 +151,7 @@ if [ $FWTYPE == "dynamic" ]; then
         elif [[ $PLATFORM == AppleTV* ]]; then
             MIN_SDK="-tvos_version_min $MIN_SDK_VERSION"
         elif [[ $PLATFORM == MacOSX* ]]; then
-            MIN_SDK="-macosx_version_min $MIN_SDK_VERSION"
+            MIN_SDK="-macos_version_min $MIN_SDK_VERSION"
```

Comment:
👀

Reply:
Yes, something changed. It works. I tested, and I will test again.